utils/db.js

```diff
@@ -42,13 +42,14 @@ const Claim = sequelize.define('claim', {
     issueUrl: Sequelize.STRING,
     pullUrl: {type: Sequelize.STRING, unique: true},
     repo: Sequelize.STRING,
+    reason: Sequelize.STRING,
```

Comment:
please add a migration. 
best place would be /db/migrations/<version>.sql
which also means, update app version on package.json